.github/workflows/vmtests.yml

```diff
@@ -65,11 +65,16 @@ jobs:
         fail-fast: false
         matrix:
            kernel:
-              - 'bpf-next-main'
-              - '5.15-main'
-              - '5.10-main'
-              - '5.4-main'
-              - '4.19-main'
+              # renovate: datasource=docker depName=quay.io/lvh-images/kind
+              - 'bpf-next-20230912.113936'
+              # renovate: datasource=docker depName=quay.io/lvh-images/kind
+              - '5.15-20230912.113936'
+              # renovate: datasource=docker depName=quay.io/lvh-images/kind
+              - '5.10-20230912.113936'
+              # renovate: datasource=docker depName=quay.io/lvh-images/kind
+              - '5.4-20230912.113936'
+              # renovate: datasource=docker depName=quay.io/lvh-images/kind
+              - '4.19-20230912.113936'
```

Comment:
We are  different than cilium (for the moment) in that we do not use the kind image. I think we should change this and use the kind image everywhere, but until we do, this should be quay.io/lvh-images/kernel-images instead of /kind.

Reply:
see: https://github.com/cilium/tetragon/blob/main/tests/vmtests/fetch-data.sh for more details

Reply:
so I changed it, in the end there is no difference since they have the same update cadence and tag, and anyway the workflow is pulling the right thing, this comment is just for renovate. But let's be more correct.